templates/shared/search/code/results.tmpl

```diff
@@ -0,0 +1,44 @@
+<div class="flex-text-block gt-fw">
+	{{$pageURL := print $.SourcePath "/search"}}
+	{{if and $.PageIsExploreCode $.ContextUser}}
+		{{$pageURL = print AppSubUrl "/" $.ContextUser.Name "/-/code"}}
+	{{else if $.PageIsExploreCode}}
+		{{$pageURL = print AppSubUrl "/explore/code"}}
+	{{end}}
```

Comment:
That's really tricky. These "if" blocks might be broken in the future and difficult to test.

Could it make the caller prepare the "PageURL" ahead?

Reply:
I will check again what can be done with this template....

Reply:
Turned out this template voodoo wasn't needed :laughing: Removed completely and replaced with a simple $.Link call

Reply:
> The parent template, or the Go code?

 The parent template, or the Go code?

There is a `.Link` variable in every parent template that gives you the current relative URL as we always tass through the page object to that code search template, we can simply use it.